packages/client/src/lib/util.test.ts

```diff
@@ -0,0 +1,30 @@
+import { formatPhoneNumber } from '../../../common/util'
+
+describe('formatPhoneNumber', () => {
+  const validPhones = [
```

Comment:
Test the case for an extension as documented in the election official `README.md`.  It should output "(123)-456-7890 Ext. 123"